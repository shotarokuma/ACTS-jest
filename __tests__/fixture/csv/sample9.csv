Column1	Column2	Column3	Column4	Column5	Column6	Column7	Column8	Column9	Column10	Column11	Column12	Column13	Column14	Column15	Column16	Column17	Column18	Column19	Column20	Column21	Column22	Column23	Column24	Column25	Column26	Column27	Column28	Column29	Column30
60	true	81	87	77	78	4	31	true	false	29	true	40	true	true	64	true	46	false	62	54	26	57	false	false	true	19	false	false	30	5	true	false	32	true	false	false	true	false	93