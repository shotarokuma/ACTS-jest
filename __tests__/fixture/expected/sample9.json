[
{"0":60,"1":true,"2":81,"3":87,"4":77,"5":78,"6":4,"7":31,"8":true,"9":false,"10":29,"11":true,"12":40,"13":true,"14":true,"15":64,"16":true,"17":46,"18":false,"19":62,"20":54,"21":26,"22":57,"23":false,"24":false,"25":true,"26":19,"27":false,"28":false,"29":30,"30":5,"31":true,"32":false,"33":32,"34":true,"35":false,"36":false,"37":true,"38":false,"39":93}
]
